src/newSeed/baseStage.scss

```diff
@@ -530,5 +530,11 @@
         padding-left: 8px;
       }
     }
+
+    &.stage7 {
+      .linksContainer {
+        width: unset;
```

Comment:
I think the right way to fix this is to delete line  463.  Our bias should always be less code rather than more, to remove unnecessary code.